packages/p2p/src/components/orders/order-details/order-details.jsx

```diff
@@ -0,0 +1,194 @@
+import React         from 'react';
+import { Button }    from 'deriv-components';
+import { localize }  from 'deriv-translations';
+import FooterActions from 'Components/footer-actions/footer-actions.jsx';
+import './order-details.scss';
+
+const OrderInfoBlock = ({ label, value }) => (
+    <div className='order-details__info-block'>
+        <p className='order-details__info-block-label'>{ label }</p>
+        <strong className='order-details__info-block-value'>{ value }</strong>
+    </div>
+);
+
+const OrderDetailsStatusBlock = ({ order_details }) => {
+    const {
+        is_buyer,
+        is_buyer_cancelled,
+        is_buyer_confirmed,
+        is_expired,
+        is_pending,
+        is_seller_confirmed,
+    } = order_details;
+
+    return (
+        <h2 className='order-details__header-status'>
+            { is_pending && is_buyer &&
+                localize('Please pay')
+            }
+            { is_pending && !is_buyer &&
+                localize('Wait for payment')
+            }
+            { is_buyer_cancelled && is_buyer &&
+                localize('You have cancelled this order')
+            }
+            { is_buyer_cancelled && !is_buyer &&
+                localize('Buyer has cancelled this order')
+            }
+            { is_expired &&
+                localize('Cancelled due to timeout')
+            }
+            { is_buyer_confirmed && is_buyer &&
+                localize('Wait for release')
+            }
+            { is_buyer_confirmed && !is_buyer &&
+                localize('Confirm payment')
+            }
+            { is_seller_confirmed &&
+                localize('Order complete')
+            }
```

Comment:
would a `status` state be easier?

Reply:
@oskar-binary We have that in `order_details.status`, but we don't want to be doing string comparisons everywhere.